lib/utils/isStandardSyntaxRule.js

```diff
@@ -22,36 +22,13 @@ module.exports = function (rule) {
 		return false;
 	}
 
-	// Called Less mixin (e.g. a { .mixin() })
-	// @ts-expect-error -- TODO TYPES support LESS and SASS types somehow
-	if (rule.mixin) {
-		return false;
-	}
-
-	// Less detached rulesets
-	if (selector.startsWith('@') && selector.endsWith(':')) {
-		return false;
-	}
-
-	// Ignore Less &:extend rule
-	if ('extend' in rule && rule.extend) {
-		return false;
-	}
-
-	// Ignore mixin or &:extend rule
-	// https://github.com/shellscape/postcss-less/blob/master/lib/less-parser.js#L52
-	// @ts-expect-error -- TODO TYPES support LESS and SASS types somehow
-	if (rule.params && rule.params[0]) {
-		return false;
-	}
-
 	// Non-outputting Less mixin definition (e.g. .mixin() {})
 	if (selector.endsWith(')') && !selector.includes(':')) {
 		return false;
 	}
 
-	// Less guards
-	if (/when\s+(not\s+)*\(/.test(selector)) {
+	// Less Parametric mixins (e.g. .mixin(@variable: x) {})
+	if (/\(@.*\)$/.test(selector)) {
 		return false;
 	}
```

Comment:
[suggestion] Does it make sense to move all the checks for the `selector` variable into `isStandardSyntaxSelector()`? If possible, it seems that `isStandardSyntaxRule()` could be simpler like this:

```js
module.exports = function (rule) {
	// ...

	if (!isStandardSyntaxSelector(rule.selector)) {
		return false;
	}

	return true;
}
```

Reply:
Done.
Thank you!